.circleci/config.yml

```
@@ -494,3 +511,32 @@ workflows:
               ignore: /.*/
             tags:
               only: /^\d+\.\d+\.\d+$/
+
+  run-windows-e2e:
```

```suggestion
  run-e2e-windows:
```

For consistency witht he label `run-e2e-windows`, can we rename this and the other instances of `run-windows-e2e`?

---

I've just realised that `run-windows-e2e` is already defined like this in the existing pipeline, not worth renaming everything perhaps, so ignore the previous comment